### ISSUE TITLE
Sehr viele Produkte fehlen in der Fertigung

### ISSUE THREAD

**Issue by jonesXYZ** [mexyn/statev_v2_issues#4120]
**Character Name**

Egal

**Auftrittszeitpunkt (Datum / Uhrzeit)**

Dauerhaft

**Beobachtetes Verhalten**

In der Fertigung fehlen einen haufen verschiedene Sachen die man nicht mehr herstellen kann wie z.b. Verpackter Strahler, Digitalkamera etc. (kp wie viele und welche genau fehlen, sie fehlen nur.

**Erwartetes Verhalten**

Produkte wiueder da. 

**Schritte um den Fehler nachvollziehen zu können**

Sollte logisch sein

**Monitorauflösung (nur wenn Falsche Darstellung in der UI)**

**Optional: Video / Bilder des Fehlers**

**Comment by TheOutburst** [comment]
Stimmt, der Strahler ist unter die Räder gekommen, next update fixed

Die Digitalkamera ist anderweitig produzierbar, da war die Dopplung eine von denen, die redundant waren, was vor paar Updates im Changelog war.

Ich kann auch sonst keine 24/7 oä Produktion finden, wo das verpackte nicht produzierbar ist, falls ich was überseheen haben sollte, gern nochmal mitteilen
